test/support/test_config.h

```diff
@@ -41,9 +41,9 @@
     (!PSTL_USE_DEBUG && (__linux__ || __APPLE__) && __INTEL_COMPILER == 1900)
 // ICC 19 generates wrong result with UDS on Windows
 #define _PSTL_ICC_19_TEST_SIMD_UDS_WINDOWS_RELEASE_BROKEN (__INTEL_COMPILER == 1900 && _MSC_VER && !_DEBUG)
-// ICC generates wrong result with UDS on MacOS
-#define _PSTL_ICC_TEST_SIMD_UDS_MACOS_RELEASE_BROKEN                                                                 \
-    (/*__INTEL_COMPILER == 1900 &&*/ !_DEBUG && __APPLE__)
+// ICC generates wrong result with UDS on icpc - 2021.5.0

```

Comment:
I suggest writing more details here - I mean "ICPC compiler generates wrong "openMP simd" code for a user defined scan operation (UDS) for MacOS, Linux and Windows"

Reply:
Done